### ISSUE TITLE
deployment-config.json file uploading on saved

### ISSUE THREAD

**Issue by tutkun** [amoussard/sftp-deployment#130]
Hi, deployment-config.json file uploading on saved. But didn't it.

**Comment by ojgarciab** [comment]
I agree, deployment-config.json MUST NOT be uploaded to remote server because security reasons (or allow empty password and ask for it in every session).

**Comment by guillobits** [comment]
UP, news about a fix ?

**Comment by harry61286** [comment]
someone fix it?
